README.md

```diff
@@ -270,30 +270,65 @@ Need help or want to contribute ideas or code? Head over to our [CONTRIBUTING](C
 
 ## Note for maintainers
 
+* Commits && Releases && Publishing
+
 All the commits in this repo follow the [Conventional Commits spec](https://www.conventionalcommits.org/en/v1.0.0/#summary). When merging a PR, make sure 1/ to
 use squash merge and 2/ that the title of the PR follows the Conventional Commits spec.
 
-The history of commits will be used to generate the `CHANGELOG`. To do so, run `yarn deploy` on the master
-branch. This command will look at all the commits since the latest tag, bump the package version according
-to semver rules, and generate a new `CHANGELOG`.
-
-If you don't want to follow semver or need to do a dry run, consult the [`standard-version` CLI usage](https://github.com/conventional-changelog/standard-version#cli-usag)
-docs. Flags for `standard-version` can be passed to `yarn deploy`.
-
-`yarn deploy`, which only does local operations and doesn't push anything, will output more or
-less the following lines:
-
-``` bash
-$ yarn deploy
-yarn run v1.21.1
-$ yarn build && standard-version -r minor
-$ rimraf lib/ && tsc
-✔ bumping version in package.json from 0.18.1 to 0.18.2
-✔ outputting changes to CHANGELOG.md
-✔ committing package.json and CHANGELOG.md
-✔ tagging release v0.18.2
-ℹ Run `git push --follow-tags origin master && npm publish` to publish
+### Preparation
+
+1. Checkout a branch `name-v5-0-1`. When deciding what version will be released it is important to look over the past commits since the last release and the updated dependencies release notes in step 2 to see if this will be a major, minor, or patch release. If you have any questions feel free to ask the maintainers.
```

Comment:
```suggestion
1. Checkout a branch `name-v5-0-1`. When deciding what version will be released it is important to look over 1) PRs since the last release and 2) release notes for any updated polkadot-js dependencies as they may affect type definitions.
```